### ISSUE TITLE
[Mobile App Bug]: Google Playstore Issue

### ISSUE THREAD

**Issue by ssandino** [socialincome-san/public#815]
We launched V1 of the app (com.socialincome.app) some time ago. It was mostly used by the beta testers while we developed. Then we decided to go with open sourced code and started developing V2 version. We also took the opportunity to change app identifier to org.socialincome.app to make sure that we can link it with our website.

We have rolled out V2, keeping the same name as for V1. Even though V2 was live, we missed a deadline for some new Google policy or SDK version deadline with V1, and V1 got suspended and get the removed by Google flag.

What's the issue we are facing? V2 doesn’t show up when you search for Social Income in the app store.

Because V1 and V2 are on the same Developer account and have the same name we suspect that it can be the problem with V1 which result in some shadow ban for search of Social Income apps or Social Income as a term.
We want to delete V1 since it’s no longer in use and it’s causing issues, but we can’t due to the removed by Google flag. It is not in the Google Play anymore, but it is still on our developer account.

Solutions: 
- finding someone at Google that can help us

**Comment by KarinBerg** [comment]
Hey @ssandino, 
I'm Karin, a friend of Verena and interested in supporting your flutter development. :) 

Is it possible to release an updated version of v1 or is this also blocked by Google?

We could create a simple version of v1 which applies to all policies and just show a screen which tells the user to use the v2 app. If this app gets reviewed and published by Google, it should remove the "removed by Google" flag and therefore the search should work again. Afterwards we can remove the v1 from the store.

But this is just a best guess. No guarantee if this is really working. But if you do not find someone at Google to help you with this problem, it could be the only chance.

**Comment by ssandino** [comment]
### Feedback from Google

Status: App not available on Google Play

We’ve reviewed your appeal request and found that your app, Social Income (com.socialincome.app) still violates Google Play policy. We’ve included details below about the specific issue with your app and what you can do to get your app back on Google Play.

For more information, you can review our e-learning course on Login Credentials before submission.

Issue: Invalid or incomplete login credentials

As explained in the Play Console Requirements policy, in order for us to review your app for compliance with Developer Program policies, you need to provide valid login credentials (such as an active demo account) and any other resources needed to review your app (for example, a QR code, etc.) via Google Play Console "App access" section.

We could not access the in-app content with the login credentials you provided. This may be due to an app connection error, or the provided credentials might not be valid. Ensure your app can be accessed for review.
You can refer to the attached screenshot for additional information. 

Kindly note that, we can process an app deletion only if the app meets all of the following criteria:

- App must be in good standing (not rejected, removed, or suspended)
- App must have 0 lifetime installs
- App must not be in review
- App must be unpublished for 24 hours 

Next steps: Provide login credentials and instructions for app access via Google Play Console

Follow these steps to submit valid login credentials for your app:

- Provide login credentials and access details for your app using the instructions described in this Play Console Help Center article under the “App access” section.
- If your app typically requires a 2-step verification code or one-time password, provide us with reusable login credentials that do not expire through Google Play Console. Alternatively, please provide complete instructions via Google Play Console to access your app (for example, email account details where we can receive a 2-step verification code).
- If your login credentials are not numeric or alphanumeric (for example, a QR code or barcode), please generate a static URL and upload it to Google Play Console.
- Resubmit your app for review after double-checking that it is compliant with the Developer Program policies, including the Play Console Requirements.
- If your Update status is "Ready to send for review" or if your Publishing overview has a section for "Changes ready to send for review", then you must click Send for review from the Publishing overview page to submit your changes for review.

The fastest way to resolve this issue and get your app back on Google Play is to follow the instructions provided above. Do not contact our policy support team to provide this information, as this can delay the review process.

Note that your changes aren't sent for review automatically. You must go to the Publishing overview page and click Send for review to submit your changes.

**Comment by ssandino** [comment]
@KarinBerg nice to meet you! Thanks for your support.

We haven't been able to solve it with Google yet (copied their answer above). I think your approach is definitely worth trying—it's something @MDikkii mentioned in the past, if I recall correctly. I'm looping him into this discussion since he worked with V1. Should we give this approach a try?

**Comment by KarinBerg** [comment]
oh ok, I see. There was only an issue with not working login credentials. I assume that the V1 app no longer works anyway, right? So, it is no option to provide working credentials with an updated v1 app.

Then, I guess the best chance is to strip down the V1 app to just on screen which tells the users to use the new V2 app. Maybe with a link to the new app in the PlayStore. 

@MDikkii What do you think? Was that your plan anyway?

Let me know, if I can support. In the meantime, I look for another task 🙂.

**Comment by MDikkii** [comment]
I'm afraid that Google doesn't like "no functionality" apps.
I thought that maybe we can just push current app with changed app id to what we had in V1. But it will not work with the firebase out of the box. So... Maybe it is worth to try one screen option first 👍

**Comment by ssandino** [comment]
Great. @KarinBerg want to take a shot at this? We'll be supporting along the way.

**Comment by KarinBerg** [comment]
@MDikkii Your concerns are valid, but I don't think we're messing anything up with this attempt. Either it works or we have to get the current app running with the old package id. If it's ok with you, I'll try to knit together a simple app 🙂.

**Comment by KarinBerg** [comment]
ok, this would be my first shot.

![Screenshot_20240621_135136](https://github.com/socialincome-san/public/assets/20112746/4c9003ac-a928-46fb-9abb-bde8d83aba7b)

Design and text is open for discussion :) 

App source:
[social_income_app_v1.zip](https://github.com/user-attachments/files/15928129/social_income_app_v1.zip)

**Comment by KarinBerg** [comment]
We released the new update for the old app and Google accepted the update and resolved all open issues.

**Comment by ssandino** [comment]
Wow this is fantastic! Great work @KarinBerg. 

Now this should solve the policy violation (Issue: Invalid or incomplete login credentials). The question is now, does this also solve the "shadow ban" in the search store – meaning our app doesn't show up when searching for it. I guess we find out in the coming days, if this changes.

**Comment by KarinBerg** [comment]
I think the issue is now solved. See screenshot :)

![Image](https://github.com/user-attachments/assets/b27c437c-cc79-4f56-a3b1-acfd9e9efd57)
